API.md

```diff
@@ -244,16 +244,48 @@ var ids = Draw.set({type: 'FeatureCollection', features: [{
 
 ### `.trash() -> Draw`
 
-This envokes the current modes trash event. For the `simple_select` mode this deletes all active features. For the `direct_select` mode this deletes the active vertices. For the drawing modes, these cancel the current process.
+This invokes the current modes trash event. For the `simple_select` mode this deletes all active features. For the `direct_select` mode this deletes the active vertices. For the drawing modes, this cancels the current process.
 
-This is different from `delete` or `deleteAlll` in that it follows rules described by the current mode.
+This is different from `delete` or `deleteAll` in that it follows rules described by the current mode.
+
+---
+
+### `.combineFeatures() -> Draw`
+
+This invokes the current modes combineFeatures event. For the `simple_select` mode this function will combine all single features into a multifeature, so long as they are all of the same geometry type. For example:
```

Comment:
Ah, single is a bit confusing. My question was cleared up by the next line, so lets drop `single`.